kong/plugins/jwt/schema.lua

```diff
@@ -18,5 +26,20 @@ return {
     claims_to_verify = {type = "array", enum = {"exp", "nbf"}},
     anonymous = {type = "string", default = "", func = check_user},
     run_on_preflight = {type = "boolean", default = true},
+    maximum_expiration = {type = "number", default = -1, func = check_maximum_expiration_positive},
   },
+  self_check = function(schema, plugin_t, dao, is_update)
+    if plugin_t.maximum_expiration ~= nil and plugin_t.maximum_expiration > 0 then
+      local has_exp = false
+      if plugin_t.claims_to_verify then
+	for index, value in ipairs(plugin_t.claims_to_verify) do
+	  has_exp = has_exp or value == "exp"
+	end
+      end
+      if not has_exp then
+	 return false, Errors.schema "when you specify maximum_expiration, 'exp' must be in claims_to_verify"
```

Comment:
style: indentation is off here and above in the same function

Reply:
Trying to come up with a more standard/consistent error message:
```
"claims_to_verify must contain 'exp' when specifying maximum_expiration"
```

Reply:
changed to "To set maximum_expiration, you need to add 'exp' in claims_to_verify"